react/features/filmstrip/reducer.ts

```diff
@@ -301,9 +303,20 @@ ReducerRegistry.register<IFilmstripState>(
                 }
             };
         case SET_VISIBLE_REMOTE_PARTICIPANTS: {
-            const { endIndex, startIndex } = action;
+            let { endIndex, startIndex } = action;
             const { remoteParticipants } = state;
-            const visibleRemoteParticipants = new Set(remoteParticipants.slice(startIndex, endIndex + 1));
+            let visibleRemoteParticipants = new Set(remoteParticipants.slice(startIndex, endIndex + 1));
+
+            // RN > 0.73 made us add this:
+            // Hack alert for iOS where we force visible remote participants
+            // _onViewableItemsChanged gets set to >= 1 when appState transitions from background to active.
+            if (navigator.product === 'ReactNative' && Platform.OS === 'ios') {
+                if (remoteParticipants.length > 1 && visibleRemoteParticipants.size >= 1) {
```

Comment:
As I expected, the problem persisted with more than one participant.

Reply:
Then this approach won't work because we'd essentially request all videos.

Imagine a 500 person conference,

Reply:
Forgot about the 500. You're right.